manifests/init.pp

```diff
@@ -13,6 +13,8 @@
   String $conf_d,
   String $confdir,
   String $config_file,
+  Optional[String] $domain_insecure,
```

Comment:
this should be an array so one can provide multiple domains

Reply:
That also sounds like a good change.